packages/rocketchat-videobridge/client/public/external_api.js

```diff
@@ -0,0 +1,377 @@
+        if(height)
+            this.iframeHolder.style.height = height + "px";
+        this.frameName = "jitsiConferenceFrame" + JitsiMeetExternalAPI.id;
+        this.url = "https://" + domain + "/";
+        if(room_name)
+            this.url += room_name;
+        this.url += "#external=true";
+
+        var key;
+        if (configOverwrite) {
+            for (key in configOverwrite) {
+                if (!configOverwrite.hasOwnProperty(key) ||
+                    typeof key !== 'string')
+                    continue;
+                this.url += "&config." + key + "=" + configOverwrite[key];
+            }
+        }
+
+        if (interfaceConfigOverwrite) {
+            for (key in interfaceConfigOverwrite) {
+                if (!interfaceConfigOverwrite.hasOwnProperty(key) ||
+                    typeof key !== 'string')
+                    continue;
+                this.url += "&interfaceConfig." + key + "=" +
+                    interfaceConfigOverwrite[key];
+            }
+        }
+
+        JitsiMeetExternalAPI.id++;
+
+        this.frame = document.createElement("iframe");
+        this.frame.src = this.url;
+        this.frame.name = this.frameName;
+        this.frame.id = this.frameName;
+        this.frame.width = "100%";
+        this.frame.height = "100%";
+        this.frame.setAttribute("allowFullScreen","true");
+        this.frame = this.iframeHolder.appendChild(this.frame);
+
+
+        this.frameLoaded = false;
+        this.initialCommands = [];
+        this.eventHandlers = {};
+        this.initListeners();
+    }
+
+    /**
+     * Last id of api object
+     * @type {number}
+     */
+    JitsiMeetExternalAPI.id = 0;
+
+    /**
+     * Sends the passed object to Jitsi Meet
+     * @param object the object to be sent
+     */
+    JitsiMeetExternalAPI.prototype.sendMessage = function(object) {
+        if (this.frameLoaded) {
+            this.frame.contentWindow.postMessage(
+                JSON.stringify(object), this.frame.src);
+        }
+        else {
+            this.initialCommands.push(object);
+        }
+
+    };
+
+    /**
+     * Executes command. The available commands are:
+     * displayName - sets the display name of the local participant to the value
+     * passed in the arguments array.
+     * toggleAudio - mutes / unmutes audio with no arguments
+     * toggleVideo - mutes / unmutes video with no arguments
+     * filmStrip - hides / shows the film strip with no arguments
+     * If the command doesn't require any arguments the parameter should be set
+     * to empty array or it may be omitted.
+     * @param name the name of the command
+     * @param arguments array of arguments
+     */
+    JitsiMeetExternalAPI.prototype.executeCommand = function(name,
+                                                             argumentsList) {
+        var argumentsArray = argumentsList;
+        if (!argumentsArray)
+            argumentsArray = [];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'argumentsArray'.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2718213841/issues/source?bid=3199634&fileBranchId=3342079#l133)